index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <nav class ="main-nav">
+        <div class="logo">
+        <a href="#"><p>Worash Aboche</p></a>  
+        </div>
+         <a href="#"> <img src ="./images/Icon-Menu.svg" alt="menu" class="menu"> </a>
+        </nav>
```

Comment:
- [ ] Please the header section should have the margins (top, bottom, left, right) defined in the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

| Your design | Figma design |
| ------------- | --------------- |
| ![image](https://user-images.githubusercontent.com/74744787/164340004-81429248-2870-4dd1-a4ea-aebbd4c451d2.png) | ![image](https://user-images.githubusercontent.com/74744787/164340163-72a4eea9-ea18-48c8-89e5-e18c0d731a5f.png) |